src/lib.rs

```diff
@@ -611,37 +578,37 @@ impl Client {
     pub fn build_report(&self) -> ReportBuilder {
         ReportBuilder {
             client: self,
-            send_strategy: None,
         }
     }
 
-    /// Function used internally to send payloads to Rollbar as default `send_strategy`.
-    fn send(&self, payload: String) -> thread::JoinHandle<Option<ResponseStatus>> {
+    /// Function used internally to send payloads to Rollbar.
+    async fn send(&self, payload: String) -> Option<ResponseStatus> {
+        let payload_clone = payload.clone();
         let body = hyper::Body::from(payload);
         let url = std::env::var("ROLLBAR_ENDPOINT").unwrap_or_else(|_| "".to_string());
+        if url.len() == 0 {
+            println!("no rollbar endpoint, not sending payload {}", payload_clone);
+            return None;
+        }
```

Comment:
this solves the problem of panicking when there is no rollbar endpoint environment variable

Reply:
Does this mean that if we misconfigure an environment and the environment variable is missing then we will silently drop rollbar sends without any indicator?

Reply:
we had already solved this problem https://github.com/remind101/r101-hq/blob/1f795ee6652421deb3527b57288ba99470bb8abd/compose.yaml#L16

why is it panicking now?

Reply:
i spoke with josh and his suggestion is to log an error by checking the env var exists on application start, rather than logging for every panic we catch